axltempl/drupal.py

```diff
@@ -88,7 +88,9 @@ def main(
 
     if not no_install:
         if shutil.which("composer") is not None:
-            os.system("composer install -o")
+            status = os.system("composer install -o")
```

Comment:
What if we can use 
```python
status = os.system("export COMPOSER_MEMORY_LIMIT=-1; composer install -o")
```
to set composer environment variable?

Reply:
@shinde-rahul, I think it's better to run `init-drupal` with the variable set. For example, for bash

```bash
COMPOSER_MEMORY_LIMIT=-1 init-drupal --directory=d8 --core=core my/site
```